src/components/GlobalPreferences/Notifications/SubscriptionsTable.js

```diff
@@ -0,0 +1,264 @@
+  )
+  const [selectedApp, setSelectedApp] = useState(-1)
+  const onAppChange = useCallback(
+    idx => {
+      setSelectedApp(idx)
+    },
+    [setSelectedApp]
+  )
+  const events = Array.from(
+    new Set(subscriptions.map(subscription => subscription.eventName))
+  )
+  const [selectedEvent, setSelectedEvent] = useState(-1)
+
+  const onEventChange = useCallback(
+    idx => {
+      setSelectedEvent(idx)
+    },
+    [setSelectedEvent]
+  )
+  const onClearFilters = useCallback(() => {
+    setSelectedEvent(-1)
+    setSelectedApp(-1)
+    setSelectedOrganization(-1)
+  }, [setSelectedEvent, setSelectedApp, setSelectedOrganization])
+
+  const onClick = useCallback(() => {
+    setIsModalOpen(true)
+  }, [setIsModalOpen])
+
+  const onCloseModal = useCallback(() => {
+    setIsModalOpen(false)
+  }, [setIsModalOpen])
+
+  const onModalConfirm = useCallback(() => {
+    setIsModalOpen(false)
+    handleUnsubscribe()
+  }, [handleUnsubscribe, setIsModalOpen])
+
+  const filteredSubscriptions = filterSubscriptions({
+    subscriptions,
+    event: events[selectedEvent],
+    appName: subscriptionApps[selectedApp],
+    organization: organizations[selectedOrganization],
+  })
+  const theme = useTheme()
+
+  return (
+    <DataView
+      fields={[
+        {
+          label: 'Organization',
+          priority: 3,
+        },
+        {
+          label: 'App',
+          priority: 2,
+        },
+        {
+          label: 'Contract',
+          priority: 2,
+        },
+        {
+          label: 'Event',
+          priority: 1,
+        },
+      ]}
+      heading={
+        <React.Fragment>
+          <DeleteSubscriptionConfirmationModal
+            visible={isModalOpen}
+            onConfirm={onModalConfirm}
+            onClose={onCloseModal}
+          />
+          <div
+            css={`
+              height: ${9 * GU}px;
+              display: flex;
+              align-items: center;
+              justify-content: space-between;
+            `}
+          >
+            <div
+              css={`
+                color: ${theme.content};
+                ${textStyle('body1')}
+              `}
+            >
+              Subscriptions
+            </div>
+            {selectedSubscriptions.length > 0 && (
+              <div>
+                <Button disabled={false} onClick={onClick}>
+                  {isSubmitting ? (
+                    <LoadingRing
+                      css={`
+                        margin-right: ${GU}px;
+                      `}
+                    />
+                  ) : (
+                    <IconMail
+                      css={`
+                        color: ${theme.negative};
+                        margin-right: ${GU}px;
+                      `}
+                    />
+                  )}{' '}
+                  Unsubscribe
+                </Button>
+              </div>
+            )}
+          </div>
+          <SubscriptionFilters
```

Comment:
We should have the same dropdown behaviour in the notification filters as elsewhere.

Each dropdown should have `'All'` as its first item, and we should default the filter's state to `-1` (unselected) if the first item is selected.